pulp_rpm/app/advisory.py

```diff
@@ -104,20 +104,29 @@ def resolve_advisories(version, previous_version):
             elif advisory_id in previous_advisory_ids:
                 advisory_id_conflicts["added_vs_previous"].append(advisory_id)
 
-    if advisory_id_conflicts["in_added"]:
-        raise AdvisoryConflict(
```

Comment:
do we need to add some user facing docs about this change?

Reply:
We talk about advisory-resolution here : https://docs.pulpproject.org/pulp_rpm/workflows/manage.html#advisory With thjis change, we're enforcing what we say there at sync-time. I can def add a sentence along the lines of "this includes when syncing from a remote that has multiple advisories with the same ID", wdyt?